libs/blocks/z-pattern/z-pattern.css

```diff
@@ -41,3 +41,58 @@
     max-width: 1200px;
   }
 }
+
+/* Alternating Z-Pattern riverflow with Ambient Video */
+.ambient-video > video {
+  object-fit: fill;
+}
+
+@media screen and (min-width: 600px) {
+  .z-pattern.alternating > .media:nth-child(odd) > .container > .media-row {
+    flex-direction: row-reverse;
+  }
+
+  .z-pattern.alternating > .media:nth-child(even) .ambient-video{
+    text-align: right;
+  }
+}
+
+/* Desktop: Ambient Video */
+@media screen and (min-width: 1024px) {
+  .ambient-video > video {
+    width: 700px;
+    height: 640px;
+  }
+}
+
+/* Tablet (Portrait Orientation): Ambient Video */
+@media screen and (min-width: 768px) and (max-width: 1024px) and (orientation: portrait) {
+  .media > .foreground .media-row > div {
+    width: auto;
+  }
+
+  .ambient-video > video {
+    width: 320px;
+    height: 292px;
+  }
+}
+
+/* Tablet (Landscape Orientation): Ambient Video */
+@media screen and (min-width: 768px) and (max-width: 1024px) and (orientation: landscape) {
+  .media > .foreground .media-row > div {
+    width: auto;
+  }
+
+  .ambient-video > video {
+    width: 480px;
+    height: 439px;
+  }
+}
+
+/* Mobile: Ambient Video */
+@media screen and (max-width: 767px) {
+  .ambient-video > video {
+    width: 100%;
+    height: auto;
+  }
+}
```

Comment:
Not sure about the media query breakpoints here. The most commonly used breakpoints for media queries in Milo are `600 900 1200 and 1440`. `768` and `1024` seem very specific to device widths. @ryanmparrish @rgclayton is this something we should standardize throughout Milo? I am assuming those are consonant breakpoints.

Reply:
Thats correct. We should probably keep to these unless we have a reason to need other ones. 

```
/* mobile only */
@media (max-width: 600px) {}

/* Tablet up */
@media screen and (min-width: 600px) {}

/* desktop up */
@media screen and (min-width: 1200px) {}

/* desktop large < 1440 */
@media screen and (min-width: 1440px) {}
```

Reply:
It might be a good idea to move these media query values into the --variable, but maybe thats for another day. :)

Reply:
Ah, I was just thinking the value. So 
```
@media screen and (min-width: var(--breakpoint-large)) {}
```
Looks kind of ugly and unnecessary though.

Reply:
I'll remove the `.ambient-video` CSS on z-pattern.css